src/graphql/types/Mutation/updateAgendaItem.ts

```
@@ -14,284 +14,281 @@ const mutationUpdateAgendaItemArgumentsSchema = z.object({
-			}),
-		},
-		description: "Mutation field to update an agenda item.",
-		resolve: async (_parent, args, ctx) => {
-			if (!ctx.currentClient.isAuthenticated) {
-				throw new TalawaGraphQLError({
-					extensions: {
-						code: "unauthenticated",
-					},
-				});
-			}
+export async function updateAgendaItemResolver(_parent: unknown, args: unknown, ctx: any){
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)